### DISCUSSION TITLE
Best practices on Nextjs app dockerization for production usage

### DISCUSSION THREAD

**Discussion by chenrui333** [vercel/next.js#21629] [Help]
👋  I am trying to dockerize my nextjs app for production usage, but I did not find many production dockerization documentation on the official site, is there any recommended best practices on it? Thanks!

**Comment by kennethnwc** [comment]
I am currently running a production site using [this](https://gist.github.com/kennethnwc/efc81d448a6381f07fd42b4305f12f68) by me.
But I would like to have a guide too.
